services/userlog/pkg/service/conversion.go

```diff
@@ -355,8 +355,10 @@ func composeMessage(nt NotificationTemplate, locale string, path string, vars ma
 	}
 
 	message, err := executeTemplate(messageraw, vars)
+	if err != nil {
+		return "", "", "", "", err
+	}
```

Comment:
No need for this `err` check. The `err` will be returned anyways one line below.